server/mailet/mailets/src/test/java/org/apache/james/transport/mailets/remote/delivery/RemoteDeliveryTest.java

```diff
@@ -21,14 +21,10 @@
 
 import static org.assertj.core.api.Assertions.assertThat;
```

Comment:
why it would be an improvement ? usually we prefer using Memory implementation for our test. Why it's different here ?